p2p/src/authenticated/mod.rs

```diff
@@ -423,7 +423,7 @@ mod tests {
 
     #[test_traced]
     fn test_determinism_one() {
-        for i in 0..10 {
+        for i in 0..1 {
```

Comment:
I think this will only run once?

Reply:
whoops yeah I was debugging an issue at some point long ago